README.md

```diff
@@ -4,3 +4,84 @@
 [![Build Status](https://travis-ci.org/GitbookIO/slate-sugar.png?branch=master)](https://travis-ci.org/GitbookIO/slate-sugar)
 
 Set of Slate helpers to create nodes and documents.
+
+## Install
+
+```
+yarn add slate-sugar
+```
+
+## Motivation
+
+The purpose of slate-sugar is to make Slate nodes and documents creation:
+
+* Painless by using smart defaults and inferring properties based on the input
+* Comprehensible by offering an API that look like a structured document
+
+To do so, it follows the hyper script convention as closely as possible:
+
+```
+(type, [props], [children]) => node
+```
+
+The reference implementation can be found in the [React documentation](https://facebook.github.io/react/docs/react-without-jsx.html) which also points to [react-hyperscript](https://github.com/mlmorg/react-hyperscript).
+
+## Simple Usage
```

Comment:
It might be nice to have a JSX example in the README with the comment config.

Also change `Simple Usage` in `Usage`.